build.gradle

```diff
@@ -69,6 +74,10 @@ jar {
     from(project.projectDir) {
         include 'LICENSE'
     }
+    from("src/main/resources/") {
```

Comment:
if the file is already under src/main/resources, isn't it automatically included in the jar file?